packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -176,7 +210,7 @@ export interface BackendDefaults {
    *
    * @default none
    */
-  readonly clientPolicy?: ClientPolicy;
+  readonly clientPolicy?: TlsClientPolicy;
```

Comment:
This shouldn't be called `clientPolicy` anymore, but `tlsClientPolicy`

Reply:
This was my careless mistake. Thanks for pointing out